Resources/Prototypes/_LostParadise/Loadouts/Items.yml

```
@@ -0,0 +1,8 @@
+# - < DarkSide - LOST PARADISE
+- type: loadout
+  id: LPPLoadoutPaperIdCardUSSP
+  category: Items
+  cost: 3
+  items:
+    - LPPPaperIdCardUSSP
+# - > 
```

yamllint

[warning] comment not indented like content

(comments-indentation)